modules/health_quest/lib/tasks/lighthouse.rake

```diff
@@ -0,0 +1,212 @@
+# frozen_string_literal: true
+
+# Rake tasks to create and update questionnaires in PGD
+
+# rubocop:disable all
+puts "IN THE #{Rails.env} ENVIRONMENT"
```

Comment:
Could you take out the puts?  It puts any time it loads.
<img width="573" alt="Screen Shot 2021-06-02 at 12 38 57 PM" src="https://user-images.githubusercontent.com/19188/120518536-ae837800-c39f-11eb-9751-23a856a76e98.png">